src/vs/languages/css/common/parser/cssSymbols.ts

```diff
@@ -102,6 +102,7 @@ export class Symbol {
 export class ScopeBuilder implements nodes.IVisitor {
```

Comment:
Thinking about this again, I think we should put all css variables always in the global scope.
That simplifies the code a bit and we get rid of the _isRootSelectorNode, which is not confusing and not correct when sass and less variables are found.